input/input.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://utilities91.netlify.app/styles.css" />
+    <link rel="stylesheet" href="./input.css" />
+    <title>Input</title>
+</head>
+
+<body>
+    <div>
+        <label for="name">Name</label>
+        <input class="py-1 px-2 text-base border rounded-sm" id="name" type="text" placeholder="Enter your name">
+    </div>
+
+
+    <div>
+        <label for="password">Name</label>

```

Comment:
label for password and you mentioned label content as Name.. is it intentional? If yes ignore the comment.